packages/find-and-replace/FindAndReplaceManager.js

```diff
@@ -209,20 +219,24 @@ class FindAndReplaceManager {
   }
 
   _propagateUpdate(renderSelection) {
+    // HACK: we make commandStates dirty in order to trigger re-evaluation
+    this._updateMarkers()
+    this.editorSession._setDirty('commandStates')
+    if (!renderSelection) {
+      this.editorSession.skipNextSelectionRerender()
```

Comment:
I would prefer to implement this in the way I explained yesterday:
- introduce a 'focused' or 'blurred' state
- skip selection re-rendering if blurred